extensions/wikia/AdEngine/js/tracking/videoInfoTracker.js

```diff
@@ -0,0 +1,51 @@
+/*global define, JSON*/
+define('ext.wikia.adEngine.tracking.videoInfoTracker',  [
+	'ext.wikia.adEngine.tracking.adInfoTracker'
+], function (tracker) {
+	'use strict';
+
+	var logGroup = 'ext.wikia.adEngine.tracking.videoInfoTracker';
+
+	function parseParameters(content, separator) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "separator". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)